drivers/scheduler/k8s/k8s.go

```diff
@@ -856,6 +910,19 @@ func (k *k8s) GetVolumeParameters(ctx *scheduler.Context) (map[string]map[string
 			result[snapData.Spec.VolumeSnapshotDataSource.PortworxSnapshot.SnapshotID] = map[string]string{
 				SnapshotParent: snap.Spec.PersistentVolumeClaimName,
 			}
+		} else if obj, ok := spec.(*snap_v1.VolumeSnapshot); ok {
```

Comment:
This PR should not have any snapshot changes. Snapshot support is already present in master

Reply:
Removed these lines code.